analyzers/tests/SonarAnalyzer.UnitTest/PackagingTests/RuleTypeTest.cs

```diff
@@ -32,29 +33,37 @@ namespace SonarAnalyzer.UnitTest.PackagingTests
     [TestClass]
     public class RuleTypeTest
     {
+        private readonly HashSet<string> deletedRules = new () {"S1145"};
```

Comment:
maybe add a comment - "rules that have been deprecated and deleted" - not to be in contradiction with the `YOU SHOULD NEVER DELETE RULES!` message